portfolio/src/main/java/com/google/sps/servlets/MyServlet.java

```diff
@@ -0,0 +1,49 @@
+package com.google.sps.servlets;
+
+// import com.google.gson.Gson;
```

Comment:
To use Gson you'll need to add it to your pom.xml file, like 


```
<dependencies>
    <!-- Gson -->
    <dependency>
      <groupId>com.google.code.gson</groupId>
      <artifactId>gson</artifactId>
      <version>2.9.0</version>
    </dependency>

// ....

```
source: JSON walkthrough

Reply:
I'm guessing it's commented out because the server wouldn't compile otherwise.